### PR TITLE
[Gen AI] Display value customization is updated to (temperature and selected model)

### PR DESCRIPTION
**Pull request by fisher-alice**
This PR adds to the model update notification the value the customization is updated to for temperature and selected model.

## Before update
Screenshot 2024-05-31 at 2 09 58 PM

## After update

Screenshot 2024-05-31 at 5 08 41 PM

## Links
[jira](https://codedotorg.atlassian.net/browse/LABS-792)

## Testing story
Tested locally in /allthethings aichat levels.

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/aichat/redux/aichatRedux.ts [hunk 1]
```diff
@@ -240,13 +240,18 @@ export const onSaveComplete =
     }
 
     changedProperties.forEach(property => {
+      const typedProperty = property as keyof AiCustomizations;
+      const propertiesSpecificityNeeded = ['temperature', 'selectedModelId'];
+      const textSuffix = propertiesSpecificityNeeded.includes(property)
+        ? ` has been updated to ${currentAiCustomizations[typedProperty]}.`
```
Comment (bencodeorg) [inline apps/src/aichat/redux/aichatRedux.ts]: It'd be slick if we could bold the value as well (I think I remember seeing this in a mock) -- I could imaging changing the type of `chatMessageSuffix` to `ReactNode` and passing in a fragment that has the content bolded. No worries if it's too much of a pain though

**Review by bencodeorg — APPROVED** [review]
Nice!

File: apps/src/aichat/redux/aichatRedux.ts (hunk 1 above)
Comment (fisher-alice) [reply apps/src/aichat/redux/aichatRedux.ts]: Chatted offline -> will ask Sam when she's back in office if she wants this specific property boldtyped. If so, wiill update `chatMessageSuffix` to be an object with an optional boldText property. Thanks!

Comment (fisher-alice) [reply apps/src/aichat/redux/aichatRedux.ts]: Updated and think it definitely looks better. Will confirm that change is preferred during next triage meeting.

File: apps/src/aichat/views/ChatMessage.tsx [hunk 2]
```diff
@@ -103,7 +103,8 @@ const displayModelUpdateMessage = (
         <>
           <span className={moduleStyles.modelUpdateMessageTextContainer}>
             <StrongText>{chatMessageText}</StrongText>
-            {chatMessageSuffix}
+            {chatMessageSuffix?.text}
+            <StrongText>{chatMessageSuffix?.boldtypeText}</StrongText>
```
Comment (bencodeorg) [inline apps/src/aichat/views/ChatMessage.tsx]: Nit: does this introduce empty elements if there's no boldtypeText? maybe `{chatMessageSuffix?.boldtypeText && <StrongText>...}`

**Review by bencodeorg — APPROVED** [review]

File: apps/src/aichat/redux/aichatRedux.ts (hunk 1 above)
Comment (fisher-alice) [reply apps/src/aichat/redux/aichatRedux.ts]: @samantha-code is good with the change. Thanks!

**Pull request merged**